src/components/CodeSnippet/CustomTheme.tsx

```diff
@@ -0,0 +1,42 @@
+export default function customTheme(): Record<string, string>{
+
+    const myCustomTheme : Record<string, string> = {
```

Comment:
you can simplify this: export default function <name> in one line
return(object)